docs/hazmat/primitives/asymmetric/serialization.rst

```diff
@@ -489,6 +489,45 @@ file suffix.
         ``additional_certificates`` is a list of all other
         :class:`~cryptography.x509.Certificate` instances in the PKCS12 object.
 
+.. function:: serialize_key_and_certificates(name, key, cert, cas,
+                                             key_encryption, cert_encryption)
```

Comment:
I think you removed `cert_encryption`?

Reply:
Paul wrote:
> I am open to opinions on whether we should have separate cert encryption and key encryption options or if we should merge the two. I think separation might be necessary for reasonable compatibility though,

https://github.com/pyca/cryptography/pull/5308#issuecomment-660531477